rclpy/src/rclpy/serialization.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLPY__SERIALIZATION_HPP_
+#define RCLPY__SERIALIZATION_HPP_
+
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+namespace rclpy
+{
+/// Serialize a ROS message
+/**
+ * Raises RCLError on serialization failure
+ * Raises TypeError if an argument has the wrong type
```

Comment:
@sloretz meta: isn't this true for any Python code, native or interpreted?

Reply:
Hmm maybe it's not true in this case since the args are both `py::object` :sweat_smile: . It's true for any method we bind with a specific type using pybind11, so probably it can be assumed rather than documented. I removed it in 3fa151c.